google/cloud/storage/tests/bucket_integration_test.cc

```diff
@@ -689,9 +669,14 @@ TEST_F(BucketIntegrationTest, NotificationsCRUD) {
                                              BucketMetadata());
   ASSERT_STATUS_OK(meta);
 
-  auto current_notifications = client->ListNotifications(bucket_name);
-  ASSERT_STATUS_OK(current_notifications);
-  EXPECT_TRUE(current_notifications->empty())
+  auto notification_ids = [&client, bucket_name] {
+    std::vector<std::string> ids;
+    auto list = client->ListNotifications(bucket_name);
+    EXPECT_STATUS_OK(list);
+    for (auto const& notification : *list) ids.push_back(notification.id());
+    return ids;
+  };
+  EXPECT_TRUE(notification_ids().empty())
```

Comment:
The message implies that this should be `ASSERT_TRUE`.

Reply:
Fixed, also searched for any other "Test aborted" and found more uses of `std::count_if()` which were easier to cleanup thanks to your new matcher.